instant-xml-macros/src/lib.rs

```diff
@@ -156,10 +160,10 @@ impl<'a> Serializer {
 #[proc_macro_derive(ToXml, attributes(xml))]
 pub fn to_xml(input: TokenStream) -> TokenStream {
     let ast = parse_macro_input!(input as syn::DeriveInput);
-
     let ident = &ast.ident;
     let root_name = ident.to_string();
     let mut output: proc_macro2::TokenStream = TokenStream::from(quote!("".to_owned())).into();
+    let mut missing_prefixes = BTreeSet::new();
```

Comment:
It's not obvious to me why we need to track missing prefixes here?

Reply:
Missing_prefix means that there is no matching prefix at current level and it needs to be looked in higher levels (parent, grand parent, etc).  Later in `fn to_xml` we are chacking if prefix exist in higher levels and currently panic if not. Missing prefixes are the ones difined on the fields, we don't need the "not missed" ones, because we can confirm on compile time that those exist. We only need "missed" to check in runtime.